src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,105 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: $c-block-bg;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+    object-fit: contain;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $c-text-primary;
+
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $c-text-secondary;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+
+    line-height: 14px;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $c-text-primary;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
```

Comment:
It's a default value ([mdn](https://developer.mozilla.org/ru/docs/Web/CSS/flex-direction))